pilot/pkg/security/trustdomain/bundle.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 var (
-	rbacLog = istiolog.RegisterScope("rbac", "rbac debugging", 0)
+	authzLog = istiolog.RegisterScope("authorization", "Istio Authorization Policy", 0)
```

Comment:
Should this be a separate logger by itself?

Reply:
yeah, probably it should be a different logger as it's under the trustdomain pkg, or maybe the better way is to remove it from the pkg directory and let the caller to log, either way it's better to be done in a separate PR.